### ISSUE TITLE
app-admin/system-config-printer instead of system-config-printer-common/gnome

### ISSUE THREAD

**Issue by and3k** [shiznix/unity-gentoo#114]
This popped up while updating my machine:

```
!!! All ebuilds that could satisfy "app-admin/system-config-printer-gnome" have been masked.
!!! One of the following masked packages is required to complete your request:
- app-admin/system-config-printer-gnome-1.4.3-r1::gentoo (masked by: package.mask)
/usr/portage/profiles/package.mask:
# Pacho Ramos <pacho@gentoo.org> (24 Jun 2015)
# Replaced by app-admin/system-config-printer, to migrate
# simply run:
# emerge -C system-config-printer-common system-config-printer-gnome
# emerge -1 system-config-printer
# Removal in a month (#553128).

(dependency required by "unity-indicators/indicator-session-12.10.5_p0_p01_p20150327-r1::unity-gentoo" [ebuild])
(dependency required by "unity-extra/unity-greeter-15.04.4_p0_p01-r1::unity-gentoo" [ebuild])
(dependency required by "x11-misc/lightdm-1.14.2_p0_p01-r3::unity-gentoo[lightdm_greeters_unity]" [ebuild])
(dependency required by "@selected" [set])
(dependency required by "@world" [argument])
For more information, see the MASKED PACKAGES section in the emerge
man page or refer to the Gentoo Handbook.
```

I guess updating the deps to app-admin/system-config-printer will fix this problem?

Thanks,
Bela

**Comment by shiznix** [comment]
Thanks should be fixed in commit 90c3a5cf94ebd4acb9b0b4adcf39e93c8b77dbe4 :)

**Comment by and3k** [comment]
Thanks
